ui/src/experiments/components/configuration/GeneralInfoConfigSection.js

```diff
@@ -50,10 +50,7 @@ export const GeneralInfoConfigSection = ({ experiment }) => {
         textStyle="reverse"
         type="responsiveColumn"
         listItems={items}
-        titleProps={{ style: { width: "30%" } }}
-        descriptionProps={{
-          style: { width: "70%", textTransform: "capitalize" },
-        }}
+        columnWidths={[1, 7/3]}
```

Comment:
Just as what had been done for Turing in https://github.com/caraml-dev/turing/pull/384/files#r1677554781, the `titleProps` and `descriptionProps` props have been replaced with `columnWidths`. `descriptionProps.style.textTransform` hasn't been working for a while now, so removing the text transform doesn't change anything.